### PR TITLE
input/access types add missing helm options

### PR DESCRIPTION
**Pull request by jensh007**
**What this PR does / why we need it**:
Add missing fields and complete documentation for helm input type

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```feature user

```

### CODE REVIEW

File: content/en/docs/guides/input_and_access.md [hunk 1]
```diff
@@ -202,6 +203,38 @@ Imports a helm chart from the local file system and adds it as a resource.
...
+spec:
+  resources:
+      ...
+  - name: mariadb-chart
+    version: 12.2.7
+    type: helmChart
+    ...
+    access:
+      ...
+      referenceName: github.com/jensh007/mariadb/charts/mariadb:12.2.7
```
Comment (reshnm) [inline content/en/docs/guides/input_and_access.md:237]: It took me some time to figure out what is described here.
So this is how the resource looks like it has been transported to an OCI-Repository.
However, shouldn't it then look something like this?

```yaml
spec:
  resources:
      ...
  - name: mariadb-chart
    version: 12.2.7
    type: helmChart
    ...
    access:
      ...
      referenceName: ghcr.io/jensh007/github.com/open-component-model/megacomponent/charts/mariadb:12.2.7
```

File: content/en/docs/guides/input_and_access.md [hunk 2]
```diff
@@ -217,15 +250,18 @@ Takes an image that is located in an OCI registry and adds it as a resource.
...
-      name: mega-image
-      version: 0.1.0
+spec:
+  resources:
+  - name: mega-image
+    version: 0.1.0
+    access:
       ...
+      referenceName: github.com/open-component-model/megacomponent/images/echo:1.10
+    ...
```
Comment (reshnm) [inline content/en/docs/guides/input_and_access.md:264]: Same as above

**Review by reshnm — CHANGES_REQUESTED** [review]

File: content/en/docs/guides/input_and_access.md [hunk 3]
```diff
@@ -202,6 +203,38 @@ Imports a helm chart from the local file system and adds it as a resource.
...
+```yaml
+  resources:
+  - name: mariadb-chart
+    type: helmChart
+    input:
+      type: helm
+      helmRepository: https://charts.bitnami.com/bitnami
+      repository: charts/mariadb
+      path: mariadb
+      version: 12.2.7
```
[the remark this answers is not in the document]
Comment (jensh007) [reply content/en/docs/guides/input_and_access.md:222]: Thanks will apply!

File: content/en/docs/guides/input_and_access.md (hunk 1 above)
Comment (jensh007) [reply content/en/docs/guides/input_and_access.md:237]: I will remove this paragraph. This section in the component-descriptor varies between the stages of transport. It is different between a cd in a CTF and in an OCI registry. So this is more confusing than helpful.

File: content/en/docs/guides/input_and_access.md (hunk 2 above)
Comment (reshnm) [reply content/en/docs/guides/input_and_access.md:251]: @jensh007 I think you should then also remove this section?

Comment (jensh007) [reply content/en/docs/guides/input_and_access.md]: done- Thank You!

**Pull request merged**